src/main/java/com/dao/homework/service/CarServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package com.dao.homework.service;
+
+import com.dao.homework.dao.CarDao;
+import com.dao.homework.dao.DriverDao;
+import com.dao.homework.lib.Inject;
+import com.dao.homework.lib.Service;
+import com.dao.homework.model.Car;
+import com.dao.homework.model.Driver;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.getById(id).get();
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        if (!driverDao.getAll().contains(driver)) {
```

Comment:
we don't need this check, when we connect a real db to our project it will handle this situation and throw an exception when a driver is not present in a db.